components/auth/AuthBox.vue

```diff
@@ -375,14 +380,25 @@ async function attemptSignup() {
     ]);
     return (loading.value = false);
   }
+  if (!turnstileToken.value || turnstileToken.value === '') {
+    signupErrors.value.record([
+      {
+        message: 'Please complete Turnstile verification',
```

Comment:
I wonder if it's clearer to phrase this as `'Please complete the captcha'` or `'Please confirm you're not a robot'` or whatever, only because I don't think the captcha box says necessarily that it's `Turnstile`, and it may not be as clear?

As ever though, absolutely fine to merge as-is, only something to change if we think is necessary.